client/src/components/FAQPanel/FAQPanel.js

```diff
@@ -0,0 +1,74 @@
+import _ from "lodash";
+import React from "react";
+
+import { SomeThingsToKeepInMind } from "src/panels/panel_declarations/common_panel_components";
+
+import common_lang from "src/panels/panel_declarations/misc/key_concept_panels/common_questions.yaml";
+
+import fin_lang from "src/panels/panel_declarations/misc/key_concept_panels/financial_questions.yaml";
+import ppl_lang from "src/panels/panel_declarations/misc/key_concept_panels/people_questions.yaml";
+import results_lang from "src/panels/panel_declarations/misc/key_concept_panels/results_questions.yaml";
+import tag_lang from "src/panels/panel_declarations/misc/key_concept_panels/tagging_questions.yaml";
+
+import sample_lang from "src/components/FAQPanel/FAQPanel.yaml";
+
+import { KeyConceptList } from "src/components/KeyConceptList/KeyConceptList";
+import { create_text_maker_component } from "src/components/misc_util_components";
+
+import { Gov } from "src/models/organizational_entities";
+
+import { secondaryColor } from "src/core/color_defs";
+
+import common_subapp_lang from "./common_faq_questions.yaml";
+import est_lang from "./estimates_comparison_questions.yaml";
+import igoc_lang from "./igoc_explorer_questions.yaml";
+import table_picker_lang from "./table_picker_questions.yaml";
+import tag_exp_lang from "./tag_explorer_questions.yaml";
+import text_diff_lang from "./text_diff_questions.yaml";
+import treemap_lang from "./treemap_questions.yaml";
+
+const { TM } = create_text_maker_component([
+  common_lang,
+  fin_lang,
+  ppl_lang,
+  results_lang,
+  tag_lang,
+  est_lang,
+  igoc_lang,
+  table_picker_lang,
+  tag_exp_lang,
+  text_diff_lang,
+  treemap_lang,
+  common_subapp_lang,
+  sample_lang,
+]);
```

Comment:
I think either it should be the consumer's responsibility to provide a `text_maker_function` to this component, or this component _and_ all the FAQ content should be lifted up to it's own directory under source (because in that case it's more of a standalone system than a reusable component).

Reply:
The former's more case-by-case boiler plate code, but it keeps this component more re-usable and is a small benefit to webpack bundling (only load the text as needed, instead of loading it all as soon as you need any FAQ).

The later is a little easier to make use of in general, and could be seen as a step towards moving FAQ text from hardcoded yaml to, potentially, something like an externally managed data source processed through the pipeline. 

I'm leaning towards the first, since moving FAQ items to pipeline data is not something we're settled on ever doing (at least not short term, the pipeline has enough work to handle).